test/conformance/runtime_contract_types.go

```diff
@@ -32,9 +32,9 @@ type MustEnvvars struct {
 
 // FilePathInfo data object returned by the environment test-image.
 type FilePathInfo struct {
-	FilePath    string `json:"FilePath"`
-	IsDirectory bool   `json:"IsDirectory"`
-	PermString  string `json:"PermString"`
+	FilePath    string `json: "FilePath"`
+	IsDirectory bool   `json: "IsDirectory"`
+	PermString  string `json: "PermString"`
```

Comment:
I think this makes `go vet` unhappy, but also this is unnecessary when they match the field names.

Reply:
Thanks for catching it.  Fixed.